### ISSUE TITLE
Update Docker image to use sonarqube 6.7.5 docker image

### ISSUE THREAD

**Issue by romani** [checkstyle/sonar-checkstyle#179]
Sonarqube allow download only for 6.7 version as LST version.

We need to start using LST version in our testing.
There is known problem with latest version of docker images - https://github.com/SonarSource/docker-sonarqube/issues/230

So for some time, we will use non-latest 6.7 instance, to avoid problems with wercker CI
https://hub.docker.com/_/sonarqube/?tab=tags

Looks like 6.7.5 is not affected by change of user - we will use it.

**Comment by romani** [comment]
instruction used: https://github.com/checkstyle/sonar-checkstyle/wiki/How-to-push-Dockerfile-to-docker-hub
6.7.5 tag is here - https://hub.docker.com/r/checkstyle/sonarqube-maven-git/tags

**Comment by romani** [comment]
wercker is failing like:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project checkstyle-sonar-plugin: There are test failures.
[ERROR] 
[ERROR] Please refer to /pipeline/source/checkstyle-sonar-plugin/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /pipeline/source/checkstyle-sonar-plugin && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -jar /pipeline/source/checkstyle-sonar-plugin/target/surefire/surefirebooter1587358871224426129.jar /pipeline/source/checkstyle-sonar-plugin/target/surefire 2018-12-30T05-55-54_269-jvmRun1 surefire3940080038367843918tmp surefire_08592442560905525173tmp
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /pipeline/source/checkstyle-sonar-plugin && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -jar /pipeline/source/checkstyle-sonar-plugin/target/surefire/surefirebooter1587358871224426129.jar /pipeline/source/checkstyle-sonar-plugin/target/surefire 2018-12-30T05-55-54_269-jvmRun1 surefire3940080038367843918tmp surefire_08592442560905525173tmp
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1
[ERROR] at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:669)
[ERROR] at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:282)
[ERROR] at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:245)
[ERROR] at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:1183)
[ERROR] at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:1011)
[ERROR] at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:857)
[ERROR] at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
[ERROR] at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
[ERROR] at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
[ERROR] at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
[ERROR] at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
[ERROR] at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
[ERROR] at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
[ERROR] at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[ERROR] at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
[ERROR] at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
[ERROR] at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
[ERROR] at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
[ERROR] at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
[ERROR] at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
[ERROR] at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] at java.lang.reflect.Method.invoke(Method.java:498)
[ERROR] at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[ERROR] at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[ERROR] at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[ERROR] at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :checkstyle-sonar-plugin
```

on wercker we do not care about validation execution ... let me disable it.

**Comment by romani** [comment]
fixes to run wercker on sonarqube 6.7.5 are merged.
